net-misc/ncp/files/ncp-1.2.4-fix-makefile.patch

```diff
@@ -0,0 +1,11 @@
+--- ncp-1.2.4/GNUmakefile	2005-05-13 19:17:17.000000000 +0200
++++ ncp-1.2.4/GNUmakefile	2020-08-18 18:11:57.113262402 +0200
+@@ -54,7 +54,7 @@
+ ncp: ncp.o libsocket
+ 	$(CC) $< -o $@ -I. $(CFLAGS) $(LDFLAGS) $(LDLIBS) `cat libsocket`
+ ifeq ($(DEBUG),)
+-	strip -R .note -R .comment ncp || strip ncp
++	#strip -R .note -R .comment ncp || strip ncp
```

Comment:
No need to comment out with a patch - we can see what we removed.

Reply:
Sorry, i don't understand? I shouldn't use a patch here? (and use `sed` instead again?)

Reply:
Sorry, what I'm saying is... if you remove a line with a patch, you can see the missing line, right?

There's no need to put a '#' in, just drop the line, because we still get to see what the old line was. :)

Reply:
ahh.. i understand now :)
i've updated the patch, now it removes the lines (and also the `ifeq .. endif` since it would be empty anyway)